### PR TITLE
Turkish Language Translation

### PR DESCRIPTION
**Pull request by TeknoCepni**
# Description
Added Turkish Language Translation for Welcome Pages

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
Could you add translation for all strings (and string arrays) for Turkish language?

**Comment by TeknoCepni** [comment]
Of course I can gladly. 🙂

**Comment by TeknoCepni** [comment]
I finished Language Translation. Please Review :)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -536,7 +536,7 @@
     <string name="choose_watermark_font_family">Choose font family</string>
     <string name="watermark_font_size">Font size</string>
     <string name="choose_watermark_style">Choose style</string>
-    <string name="viewfiles_addwatermark">1. Select the PDF you wish to add watermark to. \n2. Select Add Watermark. \n3. Fill the required information. Select OK. \n4. New PDF with watermark will be created.</string>
+    <string name="viewfiles_addwatermark">1. Select the PDF you wish to add watermark to.  2. Select Add Watermark.  3. Fill the required information. Select OK.  4. New PDF with watermark will be created.</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:539]: why did you remove '\n' ?

File: app/src/main/res/values-bs/strings.xml [hunk 2]
```diff
@@ -368,17 +360,11 @@
     <!--  Extract Text-->
     <string name="extract_text">Izdvoji Tekst</string>
     <string name="pdf_selected">Odabrana PDF datoteka: </string>
-    <string name="file_info">\n  Naziv Datoteke : %1$s \n\n  Putanja : %2$s \n\n  Veličina : %3$s \n\n  Datum Izmjene %4$s</string>
```
Comment (Swati4star) [inline app/src/main/res/values-bs/strings.xml:371]: Do we not want this translated?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -416,7 +416,7 @@
     <!-- Compress PDF -->
     <string name="compress_pdf">Compress PDF</string>
     <string name="compress_pdf_prompt">Enter % compression of PDF : </string>
-    <string name="compress_info">Congratulations! Your PDF is compressed: \n Original PDF Size : %s \n Compressed PDF Size: %s</string>
+    <string name="compress_info">Congratulations! Your PDF is compressed:   Original PDF Size : %s   Compressed PDF Size: %s</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:419]: why did you remove '\n' ?

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -367,17 +367,17 @@
     <!--  Extract Text-->
     <string name="extract_text">Extract Text</string>
     <string name="pdf_selected">PDF file selected: </string>
-    <string name="file_info">\n  File Name : %1$s \n\n  Path : %2$s \n\n  Size : %3$s \n\n  Date Modified %4$s</string>
+    <string name="file_info">"   File Name : %1$s     Path : %2$s     Size : %3$s     Date Modified %4$s"</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:370]: here also, '\n' ?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -569,9 +569,10 @@
 
     <!--  FAQs  -->
     <string name="faqSearch">Search FAQ</string>
-  <string-array name="faq_question_answers">
+    <string name="welcome_excel_message">Convert your excel files into your most desired PDFs at ease</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:572]: This string is related to welcome screen. 

Can you move it under this commment?
https://github.com/Swati4star/Images-to-PDF/blob/b775ca90cf672dac319d7116106f25db5ee2365d/app/src/main/res/values/strings.xml#L497

**Review by Swati4star — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (TeknoCepni) [reply app/src/main/res/values/strings.xml:539]: My fault, I will fix.

File: app/src/main/res/values-bs/strings.xml (hunk 2 above)
Comment (TeknoCepni) [reply app/src/main/res/values-bs/strings.xml:371]: My fault, I will add.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (TeknoCepni) [reply app/src/main/res/values/strings.xml:419]: My fault, I will fix.

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (TeknoCepni) [reply app/src/main/res/values/strings.xml:370]: My fault, I will fix.

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (TeknoCepni) [reply app/src/main/res/values/strings.xml:572]: Okay, I will do.

**Comment by TeknoCepni** [comment]
I evaluated your comments. Please Review :)
